receiver/gitproviderreceiver/metadata.yaml

```diff
@@ -31,6 +31,13 @@ attributes:
   branch.name:
     description: The name of a Git branch
     type: string
+  pull_request_state:
```

Comment:
Wouldn't `pull_request.state` be more in line with other attribute names, like `repository.name` and `branch.name`?

Reply:
They would, and that makes sense. Thanks for pointing it out!